src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(Color color, double side) {
+        super(color);
+        this.side = side;
+    }
+
+    @Override
+    public double calculateArea() {
+        return Math.round(Math.pow(side, 2));
```

Comment:
The `calculateArea` method rounds the area, which might not be necessary. Consider returning the precise value of the area without rounding, as rounding can lead to loss of precision.